### DISCUSSION TITLE
Need help with VLANs

### DISCUSSION THREAD

**Discussion by kokosowy** [gnuton/asuswrt-merlin.ng#584] [General]
Hi!

How are you doing. I use TUF-AX3000 V2 with gnuton’s firmware of course. I don’t see any tab for VLAN setup, I can’t find anything on VLANs in official AsusWRT-Merlin documentation.. Can anybody share any guide how to set up VLANs or confirm VLANs are not supported? Thanks!

**Comment by RRM5** [comment]
IF you mean the triple vlan to manage with the TUF AX3000 V2  :   LAN/IPTV
![TRIPLE VLAN](https://github.com/gnuton/asuswrt-merlin.ng/assets/15965542/9bea13a9-2a1e-4e72-983a-d7596a0b014d)

**Reply by kokosowy** [reply]
Hi @RRM5! Thanks. Hmm.. I thought this tab is meant for IPTV.. I want to isolate one router port completely from rest of network, so I could host Internet web services there. Do you think I can use it for that?

**Reply by RRM5** [reply]
I'm sorry I can't tell you what you want to do, I know what it uses to configure and access the internet, VoIP and TV.

**Reply by kokosowy** [reply]
Yeah, so I think it’s not VLAN settings I am looking for. Thank you anyway.

**Reply by Peke** [reply]
I guess you are searching something like I wrote at #536 ?

**Reply by kokosowy** [reply]
Many thanks @Peke! I will leave a comment there.
